src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,33 @@
-export const Tabs = () => {};
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            className={tab.id === selectedTabId ? 'is-active' : ''}
```

Comment:
You can create a variable for this checking `tab.id === selectedTabId` and use the, everywhere